Source/Core/Proj4Projection.js

```diff
@@ -0,0 +1,173 @@
+define([
+        '../ThirdParty/proj4',
+        './Cartesian3',
+        './Cartographic',
+        './Math',
+        './Check',
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Ellipsoid'
+    ], function(
+        proj4,
+        Cartesian3,
+        Cartographic,
+        CesiumMath,
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        Ellipsoid) {
+    'use strict';
+
+    /**
+     * MapProjection using proj4js. This projection is only to be used with Ellipsoid.WGS84.
+     * Users should exercise caution when using local-area projections, as local area projections
+     * may produce unexpected results outside their specified boundaries.
+     *
+     * Assumes Ellipsoid WGS84.
```

Comment:
Is this sentence needed?  Just above it says "This projection is only to be used with Ellipsoid.WGS84"